### DISCUSSION TITLE
Is there a plan to add parameters to the OrchardCore.Autoroute module?

### DISCUSSION THREAD

**Discussion by MikeAlhayek** [OrchardCMS/OrchardCore#6982] [General]
I need to create a route using "Autoroute" module that accepts parameters. I need the route to look something like this `/page/{id:int}/{title:slug}` 

That does not seems to work currently with Autoroute. Is there a plan on make the Autoroute module accepts parameters?

**Comment by deanmarcussen** [comment]
Duplicate of https://github.com/OrchardCMS/OrchardCore/issues/5305
